rockspec/lua-cbson-git-1.rockspec

```diff
@@ -0,0 +1,19 @@
+package = "lua-cbson"
+version = "git-1"
+source = {
+    url = "https://github.com/isage/lua-cbson/archive/master.zip",
+    dir = "lua-cbson-master"
+}
+description = {
+    summary = "Lua bindings for libbson",
+    homepage = "https://github.com/isage/lua-cbson",
+    maintainer = "isage.dna@gmail.com",
+}
+build = {
+    type = "cmake",
+    variables = {
+        LUA_INCLUDE_DIR = "$(LUA_INCDIR)",
+        LUA_LIBRARIES = "$(LUA_LIBDIR)",
+        LUA_LIBRARY = "$(LUA_LIBDIR)/lua",
```

Comment:
LUA_LIBDIR already has required path  `/usr/local/lib` e.g. But cmake cut one level to `/usr/local`. Suffix `/lua` added for this specific behavior and has no sense. But it works fine (tested on linux and macos on 5.1, luajit)

Reply:
Fix in https://github.com/bzick/lua-cbson/commit/3606e4ba7956dca14b9774138cceacf097a97701